content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Introduction.adoc

```diff
@@ -0,0 +1,30 @@
+
+== Introduction to YubiHSM 2 Windows Deployment Guide: Configure YubiHSM 2 Key Storage Provider (KSP) for Microsoft Windows Server
+
+This guide is intended to help systems administrators deploy YubiHSM 2 for use in a Windows server environment. The expected outcome is that the YubiHSM 2 is installed and configured with authentication keys, audit keys, and wrap keys. This guide also explains how to make backups and restore keys on a YubiHSM 2.
+
+These guidelines for deployment cover basic topics, so the instructions should be modified as required for your specific environment. It is assumed that you are familiar with the concepts and processes for working with Microsoft Windows Server. It is also assumed that the installation is performed on a single Microsoft Windows Server, but the concept can be extended to more servers.
+
+[IMPORTANT]
+===========
+We recommend that you use this guide for installing and testing the Microsoft Windows Server installation and setup of the YubiHSM 2 in a test or lab environment before deploying to production.
+===========
+
+
+=== About the YubiHSM Software
+
+The following YubiHSM software is used in this guide. These items are included as part of the archive file you download from the https://www.yubico.com/products/services-software/download/yubihsm-2-libraries-and-tools/[YubiHSM 2 libraries and tools page].
+
+
+=== Software
+[options="header"]
+|========================
+|Software                                                                                             |Purpose
+
+| https://developers.yubico.com/YubiHSM2/Component_Reference/yubihsm-connector/[YubiHSM Connector]    |Enables communication between the YubiHSM 2 and applications that use it. We recommend that the YubiHSM Connector run on the host operating system if the calling application is deployed to a VM. The Connector must always be running.
```

Comment:
Please make all of these links relative (like: "link:../../Component_Reference/yubihsm-connector/"

Reply:
Done